nose_randomly/plugin.py

```diff
@@ -100,13 +106,25 @@ def reset_random_seed(self):
             if have_faker:
                 faker_random.setstate(self.random_state)
 
+            if have_numpy:
+                np_random.set_state(self.random_state_numpy)
+
     @property
     def random_state(self):
         if not hasattr(self, '_random_state'):
             random.seed(self.options.seed)
             self._random_state = random.getstate()
         return self._random_state
 
+    @property
+    def random_state_numpy(self):
+        if not have_numpy:
+            raise NotImplementedError()
```

Comment:
this is probably the wrong exception to raise. I think an `ImproperlyConfigured` is likely to be the best way, plus add a message "numpy not installed"